generators/server/templates/build.gradle.ejs

```diff
@@ -243,8 +243,8 @@ configurations {
     all {
         resolutionStrategy {
             // Inherited version from Spring Boot can't be used because of regressions:
-            // https://liquibase.jira.com/browse/CORE-3536 and https://liquibase.jira.com/browse/CORE-3522
```

Comment:
These issues (https://liquibase.jira.com/browse/CORE-3536 and https://liquibase.jira.com/browse/CORE-3522) are still open. Thus are you sure it's safe to upgrade?

Reply:
This seems fixed in the changelog of 3.8.1+, but I am not sure... Even the issue with Java 11 should be fixed.